### ISSUE TITLE
CUE + LOOP Labels

### ISSUE THREAD

**Issue by Stoyvo** [Holzhaus/serato-tags#2]
It would be worth mentioning that CUE and LOOP types have labels at the end of the entries. This mean you do not need to iterate through each byte in the label field to identify the end of the label (Searching for null-terminator). Instead, utilize the data length (same level as header/data type, before data entry), and read the end of the data for X bytes.

For example:
Cues have 12 bytes before the label starts. dataLength-12 is how many bytes to read for the label. A loop has 20 bytes before the label starts (dataLength-20).

**Comment by Holzhaus** [comment]
True. If you want to change some phrasing, feel free to open a pull request.
